### PR TITLE
sveatpants

### PR DESCRIPTION
**Pull request by theotherlemonjuice**
i also renamed the folder

### CODE REVIEW

**Comment by itzTheMeow** [comment]
you seem to have ignored the folder naming instructions:
> Make sure the folder name doesn't contain spaces or special characters.

> It should also be relevant to your pack name.

**Comment by itzTheMeow** [comment]
also "now with sweatpants" doesnt really describe what it does

**Comment by theotherlemonjuice** [comment]
smh stop avoiding it

File: texturepacks/lemon's custom pack/_meta.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "lemon's custom pack",
```
Comment (itzTheMeow) [inline texturepacks/lemon's custom pack/_meta.json:2]: give it a more descriptive name

File: texturepacks/lemon's custom pack/_meta.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  "author": "lemon",
+  "description": "now with sweatpants"
```
Comment (itzTheMeow) [inline texturepacks/lemon's custom pack/_meta.json:4]: "now with sweatpants" describes what it does how?

**Review by itzTheMeow — CHANGES_REQUESTED** [review]
> **Make sure the folder name doesn't contain spaces or special characters. It should also be relevant to your pack name.**

**Comment by itzTheMeow** [comment]
there made it official

**Review by itzTheMeow — APPROVED** [review]
i guess

**Comment by theotherlemonjuice** [comment]
😤

**Pull request merged**